include/model.php

```diff
@@ -808,7 +856,7 @@ protected function update_secondary_language_terms( $slug, $name, PLL_Language $
 			}
 
 			/** @var PLL_Language $language */
-			if ( $language->slug !== $slug || $language->name !== $name ) {
+			if ( "pll_{$language->slug}" !== $slug || $language->name !== $name ) {
```

Comment:
You are fixing a bug. Could you add a corresponding test? Or fix the tests already covering this line?

Reply:
I'm not sure this worth adding a test.
The bug was only calling `wp_update_term()` with new data === old data, so it was adding unnecessary process but was doing nothing. I can still try to add one that makes sure certain actions are not triggered I guess.

Reply:
forget it.